### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by Rockyraven**
create  Grid Component
image

### CODE REVIEW

File: component/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+.aside-rt{
+    height: 10rem;
+    margin-left: 0.5rem;
+    background-color: var(--warning-color);
+    grid-area: aside-rt;
+}
+.main-content{
+    height: 10rem;
+    background-color: skyblue;
```
Comment (khushijohri2001) [inline component/grid/grid.css]: you can give a var color here and mention it in the root

**Review by khushijohri2001 — APPROVED** [review]
clean code good for merging

File: component/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    "navbar navbar navbar"
+    "aside-lt main-content aside-rt"
+    "aside-lt main-content aside-rt"
+    "aside-lt main-content aside-rt"
+    "footer-bar footer-bar footer-bar";
+}
+.flex{
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (pradipgharge) [inline component/grid/grid.css:74]: You can extract this out as a separate class, to avoid repetition

**Review by pradipgharge — APPROVED** [review]
code looks clean and good to merge !

**Pull request merged**